### ISSUE TITLE
In the released version Vomnibar doesn't show favicon on Firefox

### ISSUE THREAD

**Issue by rdewolff** [gdh1995/vimium-c#180]
The title says it all. It would be great to show the site favicons in the list.

<img width="340" alt="New_Issue_·_gdh1995_vimium-c" src="https://user-images.githubusercontent.com/130704/82745481-32feec00-9d85-11ea-9ad6-eae747231427.png">

**Comment by gdh1995** [comment]
are you using firefox? it should have shown there.
And, what's your settings of "preferred vomnibar page" option?



---Original---

**Comment by rdewolff** [comment]
Yep am using Firefox.

My Vomnibar settings are : 

```json
{
  "maxMatches": 10,
  "queryInterval": 500,
  "sizes": "77,3,44",
  "styles": "mono-url"
}
```

Preffered vomnibar page is: `front/vomnibar.html`

**Comment by gdh1995** [comment]
Confirmed. This is a regression.

Vimium C can show favicons on Chrome, or when it's in debugging mode and on Firefox.

**Comment by rdewolff** [comment]
Just tested on Chrome and this works.

**Comment by gdh1995** [comment]
This has been fixed by 0d737fb5e7ceb99f80b457b59e53faabb21e42de

**Comment by gdh1995** [comment]
v1.83.3 has been released and it fixes this.
